### PR TITLE
Fix maintainer list

### PR DESCRIPTION
**Pull request by isuruf**
cc @ArneKr, @ocefpaf 

@conda-forge/core There's a PR in conda-smithy that checks this lint https://github.com/conda-forge/conda-smithy/pull/541/files

### CODE REVIEW

File: recipes/r-diptest/meta.yaml [hunk 1]
```diff
@@ -51,4 +51,4 @@ about:
 
 extra:
   recipe-maintainers:
-    ArneKr
+    - ArneKr
```
Comment (ocefpaf) [inline recipes/r-diptest/meta.yaml]: Argh! The linter should catch that :unamused: 

Thanks @isuruf for catching and fixing it!

**Comment by jakirkham** [comment]
This will still need to be fixed in the feedstock unfortunately.

ref: https://github.com/conda-forge/r-diptest-feedstock/blob/master/recipe/meta.yaml#L54

**Comment by isuruf** [comment]
This recipe is not removed from staged-recipes yet, so it'll be fixed once the master in travis-ci is run to completion.

**Comment by jakirkham** [comment]
Unfortunately I have not found this to be the case. Once the recipe is committed in the feedstock it won't change it again from `staged-recipes` (even if the recipe is not removed from here yet). This has been the case since the beginning. I'm guessing Phil added this as some sort of fail safe to avoid weird things in staged-recipes messing up the feedstock. Though definitely keep an eye on it and if turns out I'm now wrong feel free to let me know.

**Comment by isuruf** [comment]
@jakirkham, you were right. @ArneKr can you fix this in https://github.com/conda-forge/r-diptest-feedstock ?

**Comment by ocefpaf** [comment]
No need @ArneKr. I did that in https://github.com/conda-forge/r-diptest-feedstock/pull/1 b/c you would not have merge rights without it.

**Pull request merged**